scripts/managed-services/get-kafka-instance-state/get-kafka-instance-state.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -o nounset
+set -o pipefail
+
+function main() {
+  local timestamp
+  timestamp=$(date +%s)
+  local inspectDir
+  inspectDir="inspect.${resource_namespace}.${timestamp}"
+  mkdir "${inspectDir}"
+
+  if [[ ! $(oc get namespace "${resource_namespace}" &> /dev/null) ]]; then
```

Comment:
This is the only place I think it'll contaminate the tarfile

Reply:
Should that still happen, even with the new `&> /dev/null` addition?

Reply:
I think what you are looking for is the exit code right?

Then it sohuld be `if oc get namespace "${resource_namespace}" &> /dev/null; then`

Reply:
I didn't see the `&>` when I reviewed.. I'll look again :)

Reply:
Done

Reply:
I don't think this check is working correctly

```
$ oc get namespace kafka-c6f606rg996p1r6q4kj0
NAME                         STATUS   AGE
kafka-c6f606rg996p1r6q4kj0   Active   46h
$ oc get namespace kafka-c6f606rg996p1r6q4kj0 &> /dev/null
$ echo $?
0
```
I also tried running this script using the new testing tooling and it's failing on this check even when I confirm it can see the namespace.

`if ! oc get namespace "${resource_namespace}" &> /dev/null; then` is working correctly because we want to fail out if it cannot find the namespace.

Reply:
```
 edward@derg-arch  ~  oc get ns asdf
echo $?
Error from server (NotFound): namespaces "asdf" not found
 ✘ edward@derg-arch  ~  echo $?
1
 edward@derg-arch  ~ 
```